crates/katana/core/src/service/messaging/service.rs

```diff
@@ -270,17 +270,11 @@ fn trace_msg_to_l1_sent(messages: &Vec<MessageToL1>, hashes: &Vec<String>) {
             #[rustfmt::skip]
             info!(
                 target: LOG_TARGET,
-                r"Message executed on settlement layer:
-| from_address | {}
-|  to_address  | {}
-|   selector   | {}
-|   payload    | [{}]
-
-",
-                m.from_address,
-                to_address,
-                selector,
-                payload_str.join(", ")
+                "Message executed on settlement layer",
```

Comment:
What do these look like when logged with this new format vs the old? Is it a significant reduction in human readability?

Reply:
I believe It will look like this
`INFO [messaging] Message sent to settlement layer { hash="the_hash_value", from_address="source_address", to_address="destination_address", payload="data1, data2, data3" }
`

Reply:
Ok sounds good. Let's go ahead with the change